ftplugin/purescript_vimmerps.vim

```diff
@@ -23,11 +23,11 @@ com! -buffer -nargs=1 Psearch call vimmerps#Psearch(<q-args>)
 
 if !g:vimmerps_disable_mappings
     " Keybindings for IDE like funtions
-    nm <buffer> <silent> <leader>a :Papply<CR>
-    nm <buffer> <silent> <leader>i :Pimport<CR>
-    nm <buffer> <silent> <leader>g :call LanguageClient_textDocument_definition()<CR>
-    nm <buffer> <silent> <leader>h :call LanguageClient_textDocument_hover()<CR>
-    nm <buffer> <silent> <leader>l :Pbuild<CR>
+   autocmd Filetype purescript nm <buffer> <silent> <leader>a :Papply<CR>
```

Comment:
This is not needed. This file is present in `ftplugin` and has `purescript_` in it's name. These won't be called in other buffers.

Reply:
yes @sriharshachilakapati you are right, the tag wasn't correct hence I was not able to use `help` function, now that I have updated tags, It'll work now